ap.md

```diff
@@ -0,0 +1,236 @@
+<div class="large-12 columns">
+
+<hr>
+
+Rename the “index.js” file you’ve just created to “ap.js”, then copy and paste the below script over the existing text:
+
+{% highlight javascript %}
+// Import the interface to Tessel hardware
+var tessel = require('tessel');
+// Load the http module to create an http server.
+var http = require('http');
+
+// Configure our HTTP server to respond with "Hello from Tessel!" to all requests.
+var server = http.createServer(function (request, response) {
+  response.writeHead(200, {"Content-Type": "text/plain"});
+  response.end("Hello from Tessel!\n");
+});
+
+// Listen on port 8080, IP defaults to 192.168.1.101. Also accessible through [tessel-name].local
+server.listen(8080);
+
+// Put a friendly message on the terminal
+console.log("Server running at http://192.168.1.101:8080/");
+
+{% endhighlight %}
+
+Now that our server is in place, let's get our access point set up. In the terminal, run the following command:
+
+`t2 ap -n TesselRouter`
+
+This will make an open wifi network called TesselRouter. If you open the wifi setting of your computer or a separate device like a smartphone or tablet, and scan for new devices, you should be able to see and connect to this new network. After connecting to TesselRouter, run the following command in your terminal:
+
+`t2 run ap.js`
+
+Once you see "Server running at http://192.168.1.101:8080/" in the terminal, go to http://192.168.1.101:8080/ (or http://tessel.local:8080/, replacing "tessel" with the name of your Tessel) in the web browser of the device connected to the TesselRouter network. You should "Hello from Tessel!" appear on the screen. You can press "Control+C" while in your terminal to stop this server. 
+
+Now let's take it up a notch by adding some interactivity between the web page and the Tessel!
+
+<hr>
+
+Let's start this next part by building out the web page we want Tessel to send to your web browser. Create a file called `index.html` in your project directory and open it up in your preferred text editor to add the initial html for our web page:
+
+{% highlight html %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Tessel Web App</title>
+</head>
+<body>
+  <h1>Hello from Tessel!</h1>
+  <p>Press a button to toggle an LED.</p>
+  <ul>
+    <li>
+      <button class="led-button" data-led="2">Green</button>
+      Status: <span class="led-status"></span>
+    </li>
+    <li>
+      <button class="led-button" data-led="3">Blue</button>
+      Status: <span class="led-status"></span>
+    </li>
+  </ul>
+</body>
+</html>
+{% endhighlight %}
+
+If you open that file in your browser, you should see something like this:
+
+![web app screenshot](./images/ap-web-app-preview.png)
+
+In that html, we have a heading, a line of instructions for the user, and a list of buttons for controlling a corresponding LED on the Tessel. Below that list of buttons we'll add some JavaScript for communicating with the Tessel:
```

Comment:
I would put the explanation of the HTML right below the code block containing it